Makefile

```diff
@@ -16,41 +16,41 @@ PLATFORM_ID = $$( uname -s )
 endif
 PLATFORM = $$( \
 	case $(PLATFORM_ID) in \
-		( Linux | FreeBSD | OpenBSD | NetBSD | Windows) echo $(PLATFORM_ID) ;; \
+		( Linux | FreeBSD | OpenBSD | NetBSD | Windows | Darwin ) echo $(PLATFORM_ID) ;; \
 		( * ) echo Unrecognized ;; \
 	esac)
 
 CTAGS = $$( \
 	case $(PLATFORM_ID) in \
-		( Linux ) echo "ctags" ;; \
+		( Linux | Darwin ) echo "ctags" ;; \
 		( FreeBSD | OpenBSD | NetBSD | Windows) echo "exctags" ;; \
 		( * ) echo Unrecognized ;; \
 	esac)
 
 MAKE = $$( \
 	case $(PLATFORM_ID) in \
-		( Linux | Windows) echo "make" ;; \
+		( Linux | Windows | Darwin ) echo "make" ;; \
```

Comment:
Should this be added to the next line instead to invoke ``gmake``?

Reply:
It's just `make`. I don't remember if I installed it through `brew`, but there's no `gmake` on my system.